src/sql/workbench/browser/modelComponents/listView.component.ts

```diff
@@ -193,11 +193,21 @@ interface ExtensionListTemplate {
 
 class OptionsListRenderer implements IListRenderer<ListViewOption, ExtensionListTemplate> {
 	public static TEMPLATE_ID = 'optionListRenderer';
+	private _forceIconPadding: boolean = false;
 
 	public get templateId(): string {
 		return OptionsListRenderer.TEMPLATE_ID;
 	}
 
+	public set forceIconPadding(v: boolean) {
```

Comment:
looks like all you need is a public property: `public forceIconPadding : boolean = false;`